### PR TITLE
Fixed "Resources.resx" for "RevitNodes"

### PR DESCRIPTION
**Pull request by Benglin**
This pull request attempts to do the following:

1. Align `*.resx` in all of  `Revit_2014`, `Revit_2015` and `Revit_2016` branches.
2. We do not want `DynamoRevit.csproj` to generate documentation `XML` file (because the file is used for localization process, and it is not user-facing).

Hi @Randy-Ma, could you please have a look at this? Thanks!

### CODE REVIEW

File: src/DynamoRevit/DynamoRevit.csproj [hunk 1]
```diff
@@ -36,6 +36,8 @@
     <Prefer32Bit>false</Prefer32Bit>
     <DocumentationFile>$(OutputPath)\DynamoRevitDS.XML</DocumentationFile>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DynamoRevitDS.xml</DocumentationFile>
```
Comment (Randy-Ma) [inline src/DynamoRevit/DynamoRevit.csproj]: As I can see from the summary, the xml files will not be generated. But it is not the case here.

File: src/DynamoRevit/DynamoRevit.csproj [hunk 2]
```diff
@@ -48,6 +50,8 @@
...
```
Comment (Randy-Ma) [inline src/DynamoRevit/DynamoRevit.csproj]: As I can see from the summary, the xml files will not be generated. But it is not the case here.

File: src/Libraries/RevitNodes/Properties/Resources.en-US.resx [hunk 3]
```diff
@@ -144,7 +144,7 @@
   <data name="CurtainPanelShouldRepresentRevitPanel" xml:space="preserve">
     <value>Curtain Panel should represent Revit panel</value>
   </data>
-  <data name="NumberOfDivisionsShouldBeGreaterThan2" xml:space="preserve">
+  <data name="NumberOfDivisionMustBeGreaterThan2" xml:space="preserve">
```
Comment (Randy-Ma) [inline src/Libraries/RevitNodes/Properties/Resources.en-US.resx]: NumberOfDivisionsMustBeGreaterThan2 looks better than NumberOfDivisionMustBeGreaterThan2

File: src/Libraries/RevitNodes/Properties/Resources.resx [hunk 4]
```diff
@@ -144,7 +144,7 @@
   <data name="CurtainPanelShouldRepresentRevitPanel" xml:space="preserve">
     <value>Curtain Panel should represent Revit panel</value>
   </data>
-  <data name="NumberOfDivisionsShouldBeGreaterThan2" xml:space="preserve">
+  <data name="NumberOfDivisionMustBeGreaterThan2" xml:space="preserve">
```
Comment (Randy-Ma) [inline src/Libraries/RevitNodes/Properties/Resources.resx]: NumberOfDivisionsMustBeGreaterThan2 looks better than NumberOfDivisionMustBeGreaterThan2

File: src/DynamoRevit/DynamoRevit.csproj [hunk 5]
```diff
@@ -36,6 +36,8 @@
...
+    <DocumentationFile>
+    </DocumentationFile>
```
Comment (Randy-Ma) [inline src/DynamoRevit/DynamoRevit.csproj]: The xml tag does not contain anything. we can remove it.

File: src/DynamoRevit/DynamoRevit.csproj [hunk 6]
```diff
@@ -48,6 +50,8 @@
...
```
Comment (Randy-Ma) [inline src/DynamoRevit/DynamoRevit.csproj]: The xml tag does not contain anything. we can remove it.

File: src/DynamoRevit/DynamoRevit.csproj (hunk 1 above)
Comment (Benglin) [inline src/DynamoRevit/DynamoRevit.csproj]: Argh! That's `cherry-pick` being smart on me! Okay will fix this, good catch.

File: src/Libraries/RevitNodes/Properties/Resources.en-US.resx (hunk 3 above)
Comment (Benglin) [inline src/Libraries/RevitNodes/Properties/Resources.en-US.resx]: That's a great suggestion! Updated :)

**Comment by Benglin** [comment]
Hi @Randy-Ma, those are really good catches. I have updated the PR, please have a look, thanks!

**Comment by Randy-Ma** [comment]
@Benglin 
Thanks for the fix and the update! LGTM!

**Pull request merged**